core/pom.xml

```diff
@@ -163,6 +163,7 @@ limitations under the License.
           <includes>
             <include>org/apache/calcite/test/CalciteSuite.java</include>
           </includes>
+          <argLine>-Xmx1536m -XX:MaxPermSize=256m -Duser.timezone=UTC</argLine>
         </configuration>
```

Comment:
did you intend to check this in?
* MaxPermSize is deprecated in 1.8.
* Xmx might prevent the suite from running on small machines.
* And fixing timezone reduces our test coverage